src/test/resources/output/json/00733/HT007331917.json

```diff
@@ -0,0 +1,128 @@
+      "source" : {
+        "id" : "http://d-nb.info/gnd/7749153-1",
+        "label" : "Gemeinsame Normdatei (GND)"
+      },
+      "type" : [ "Person" ]
+    },
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/ill",
+      "label" : "Illustration"
+    },
+    "type" : [ "Contribution" ]
+  } ],
+  "contributorOrder" : [ "http://d-nb.info/gnd/118527533 | http://d-nb.info/gnd/118705172" ],
+  "describedBy" : {
+    "dateCreated" : "19970502",
+    "id" : "http://lobid.org/resources/HT007331917",
+    "label" : "lobid Ressource",
+    "sourceOrganization" : {
+      "id" : "http://lobid.org/organisations/DE-836#!",
+      "label" : "lobid Organisation"
+    }
+  },
+  "extent" : "89 S. : Ill.",
+  "hasItem" : [ {
+    "callNumber" : "AQSH 1",
+    "describedBy" : {
+      "id" : "http://lobid.org/items/HT007331917:DE-836:AQSH%201",
+      "label" : "lobid Bestandsressource"
+    },
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-836",
+      "label" : "lobid Organisation"
+    },
+    "id" : "http://lobid.org/items/HT007331917:DE-836:AQSH%201#!",
+    "itemOf" : {
+      "id" : "http://lobid.org/resources/HT007331917#!",
+      "label" : "lobid Ressource"
+    },
+    "label" : "AQSH 1",
+    "note" : "00007007",
+    "type" : [ "Item" ]
+  } ],
+  "hbzId" : "HT007331917",
+  "id" : "http://lobid.org/resources/HT007331917#!",
+  "isbn" : [ "9783922406662", "3922406661" ],
+  "medium" : [ {
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010",
+    "label" : "Print"
+  } ],
+  "publication" : [ {
+    "location" : "Memmingen",
+    "publishedBy" : "Visel",
+    "startDate" : "1996",
+    "type" : [ "PublicationEvent" ]
+  } ],
+  "responsibilityStatement" : [ "Annette von Droste-Hülshoff. Holzstiche von Karl-Georg Hirsch" ],
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-HT007331917",
+    "label" : "Culturegraph Ressource"
+  } ],
+  "subject" : [ {
+    "componentList" : [ {
+      "altLabel" : [ "Hirsch, Karl Georg", "Hirsch, Karl-Georg (Künstler)" ],
+      "dateOfBirth" : "1938",
+      "id" : "http://d-nb.info/gnd/118705172",
+      "label" : "Hirsch, Karl-Georg",
+      "source" : {
+        "id" : "http://d-nb.info/gnd/7749153-1",
+        "label" : "Gemeinsame Normdatei (GND)"
+      },
+      "type" : [ "Person" ]
+    }, {
+      "id" : "http://d-nb.info/gnd/4123412-1",
+      "label" : "Illustration",
+      "source" : {
+        "id" : "http://d-nb.info/gnd/7749153-1",
+        "label" : "Gemeinsame Normdatei (GND)"
+      },
+      "type" : [ "SubjectHeading" ]
+    }, {
+      "id" : "http://d-nb.info/gnd/4125166-0",
+      "label" : "Holzstich",
+      "source" : {
+        "id" : "http://d-nb.info/gnd/7749153-1",
+        "label" : "Gemeinsame Normdatei (GND)"
+      },
+      "type" : [ "SubjectHeading" ]
+    }, {
+      "dateOfBirth" : "1797",
+      "dateOfDeath" : "1848",
+      "id" : "http://d-nb.info/gnd/4113356-0",
```

Comment:
The work shouldn't have a birth/death date. But then, it's catalogued in that way. So. Be it.